incubator/kafka/templates/statefulset.yaml

```diff
@@ -20,6 +20,25 @@ spec:
         app: {{ include "kafka.name" . | quote }}
         release: {{ .Release.Name | quote }}
     spec:
+    {{- if .Values.external.enabled }}
+      ## ref: https://github.com/Yolean/kubernetes-kafka/blob/master/kafka/50kafka.yml
+      initContainers:
+      - name: init-config
+        image: solsson/kafka-initutils@sha256:c98d7fb5e9365eab391a5dcd4230fc6e72caf929c60f29ff091e3b0215124713
```

Comment:
Is there a better image tag by any chance?

Reply:
Shouldn't the image also be configureable?

Reply:
Agreed on both points. I'll fix this ASAP.  Thanks!!!